src/hotspot/os/linux/os_linux.cpp

```diff
@@ -2801,11 +2801,6 @@ void os::pd_commit_memory_or_exit(char* addr, size_t size, bool exec,
   #define MAP_HUGE_SHIFT 26
 #endif
 
-// Define MADV_HUGEPAGE here so we can build HotSpot on old systems.
-#ifndef MADV_HUGEPAGE
-  #define MADV_HUGEPAGE 14
-#endif
-
```

Comment:
Reviewer note: I remove this since I am sure nobody builds on Linux < 2.6.38 - and if we would, probably a lot of other stuff would break first. I am unemotional though and could leave it in if desired.